mitxgraders/helpers/mitmath/mathfuncs.py

```diff
@@ -284,6 +301,21 @@ def merge_dicts(*source_dicts):
     'm': 1e-3, 'u': 1e-6, 'n': 1e-9, 'p': 1e-12
 }
 
+pauli = {
```

Comment:
Hmm. Not sure I want these variables called `x`, `y` and `z` -- too likely to clash with other variable names (is there a field that calls them x/y/z, actually? I've seen a variety of conventions, but not that). Can we do `sigma_x`, `sigma_y` and `sigma_z`? (Also happy to hear opinions.)

Reply:
That was ... silly of me. 

I don't know of any fields that call them lowercase `x`, `y`, and `z`. Capital `X`, `Y`, and `Z` are the standard in quantum information. (Well, 6 years ago anyway.)

Changed to `sigma_x`, etc. A QI course can change their names if desired.